### ISSUE TITLE
Fatal error, need to update EntityInterface entityType to be static

### ISSUE THREAD

**Issue by herbdool** [backdrop-contrib/flag#14]
Core had an API change. Need an update here like this:

```php
          /**
		   * Implements EntityInterface::entityType().
		   *
		   * @since 1.13.0 This method is now static.
		   */
		  public static function entityType() {
		    return 'entity_test';
		  }
```

**Comment by herbdool** [comment]
@jromine I've got a bug fix PR https://github.com/herbdool/flag/pull/1

**Comment by jenlampton** [comment]
I have confirmed this fix works, thanks @herbdool

**Comment by quicksketch** [comment]
This change is being reverted in core and will be fixed in 1.13.1: https://github.com/backdrop/backdrop-issues/issues/3809

**Comment by jenlampton** [comment]
Great, closing this issue. The core change will make this change unnecessary (and problematic)
